prac_05/hex_colours.py

```diff
@@ -0,0 +1,33 @@
+"""
+Hexadecimal Color Code Lookup
+Estimate: 30 minutes
+Actual:   45 minutes
+"""
+
+# Dictionary containing color names and their hexadecimal codes
+COLOR_CODES = {
```

Comment:
The name COLOR_CODES suggests a list of colour codes, a name such as COLOR_TO_CODE might work better for dictionaries?